Intro_Sort/README.md

```diff
@@ -0,0 +1,84 @@
+# Introsort
+
+1. It was invented by David Musser in 1997.
+
+2. It is also known as introspective sort.
+
+3. It is a hybrid sorting algorithm(combines two or more other algorithms that solve the same problem,either choosing one depending upon the data or switching between them over the course of algorithm).
+
+4. It provides both fast average performance and optimal worst performance,thus allowing the performance requirement to become more strict and effective.
+
+5. It combines the good part of three algorithm i.e quick sort, heap sort,insertion sort.
```

Comment:
the details is not enough. please include some more. if possible please add a GIF